uaclient/tests/test_security_status.py

```diff
@@ -11,9 +11,8 @@
 M_PATH = "uaclient.security_status."
 
 
-def mock_package(
-    name, installed=None, candidate=None, archive=None, origin=None
-):
+# Candidates are a tuple of (version, archive, origin)
+def mock_package(name, installed=None, candidates=[]):
```

Comment:
Yeah, I put the comment there exactly not to create too many structures just to mock the tests.
If you think it would be useful, then of course, can be done.

Reply:
Maybe a named tuple would be a bit much just for this, but how about something like this:
```suggestion
# Candidates are a tuple of (version, archive, origin)
def mock_package(name, installed=None, candidates: Tuple[str, str, str] = ()):
```

Reply:
just updated